DEVELOPING_OPAL/demos/src/main/scala/org/opalj/fpcf/analyses/FieldAccessInformationAnalysisDemo.scala

```diff
@@ -54,19 +60,25 @@ object FieldAccessInformationAnalysisDemo extends ProjectAnalysisApplication {
             case Some(requirements) => requirements + domain
         }
 
-        var propertyStore: PropertyStore = null
+        val propertyStore: PropertyStore = project.get(PropertyStoreKey)
         var analysisTime: Seconds = Seconds.None
         val analysesManager = project.get(FPCFAnalysesManagerKey)
+        val typeIterator = XTACallGraphKey.getTypeIterator(project)
+        project.updateProjectInformationKeyInitializationData(TypeIteratorKey) { _ => typeIterator }
+        project.updateProjectInformationKeyInitializationData(ContextProviderKey) { _ => typeIterator }
```

Comment:
I don't think it's necessary to configure both keys anymore. In fact, I think configuring the TypeIteratorKey produces an error now.

Reply:
![image](https://github.com/opalj/opal/assets/78490564/a79b39d1-c5a0-4ca5-880a-8d6d2fd699b7)

Apparently it does

Reply:
Now there's an unused import

Reply:
WIP